app/models/campaign.rb

```diff
@@ -123,6 +124,12 @@ def date_range
     "#{start_date.to_s "mm/dd/yyyy"} #{end_date.to_s "mm/dd/yyyy"}"
   end
 
+  def date_range=(value)
+    dates = value.split(" - ")
+    self.start_date = Date.strptime(dates[0], "%m/%d/%Y")
+    self.end_date   = Date.strptime(dates[1], "%m/%d/%Y")
+  end
```

Comment:
Would prefer to see this use `to_s` with one of the `config/initializers/date_formats.rb`

Reply:
This is parsing, not printing